contracts/core/GuardableModifier.sol

```diff
@@ -89,4 +89,13 @@ abstract contract GuardableModifier is Module, Guardable, Modifier {
             IGuard(currentGuard).checkAfterExecution(bytes32(0), success);
         }
     }
+
+    function _authorizer() private returns (address) {
```

Comment:
`aurthorizer` doesn't feel quite right here.
I quite like Open Zeppelin's nomenclature for this, they use `_msgSender()`. Perhaps to avoid naming conflicts with OZ, we could use something like `messageSender()` or `caller()`?

Reply:
I think we need a better name, because we are exactly decoupling the msg.sender from the authorization

Reply:
It's just an auxiliary private function. Maybe sentOrSignedBy? Other ideas?

Reply:
`sendingModule()`?

`sentOrSignedBy()` is quite good.

Reply:
sorted